playbooks/roles/ocp-config/README.md

```diff
@@ -11,32 +11,49 @@ Requirements
+| setup_squid_proxy           | no       | false                                 | Flag for setting up squid proxy server on bastion node |
+| proxy_url                   | no       | ""                                    | Proxy url eg: http://[user:passwd@]server:port (NA when setup_squid_proxy: true)|
+| no_proxy                    | no       | ""                                    | Comma seperated string of domains/cidr to exclude proxy |
+| enable_local_registry       | no       | false                                 | Set to true to enable usage of local registry for restricted network install |
+| fips_compliant              | no       | false                                 | Set to true to enable usage of FIPS for OCP deployment |
+| chronyconfig.enabled        | no       | true                                  | Set to true to enable chrony configuration on the bastion node during installation. This also configure the bastion as a NTP server for the cluster. |
+| chronyconfig.content        | no       | ""                                    | List of time NTP servers and options pair (see chronyconfig examples). If empty, bastion will try sync with some default ntp server (internet) AND local HW clock (with higher stratum). |
+| chronyconfig.allow          | no       | ""                                    | List of network cidr (X.X.X.X/Y) allowed to sync with bastion configured as NTP server |
+| dhcp_shared_network         | no       |                                       | Flag to update DHCP server work on a shared network. (Neither ACK nor NACK unknown clients) |
+| cni_network_provider        | no       | OVNKubernetes                         | Sets the default Container Network Interface (CNI) network provider for the cluster |
+| cni_network_mtu             | no       |                                       | MTU value to assign to the CNI network. Recommended values for OpenshiftSDN: <NIC MTU> - 50; OVNKubernetes: <NIC MTU> - 100 |
+| cluster_network_cidr        | no       | 10.128.0.0/14                         | Network (in CIDR) used for the pod networks.
+| cluster_network_hostprefix  | no       | 23                                    | The subnet prefix length to assign to each individual node. (netmask in CIDR format)
+| service_network             | no       | 172.30.0.0/16                         | Network (in CIDR) used for the service network.
+| rhcos_pre_kernel_options    | no       | []                                    | List of day-1 kernel options for RHCOS nodes eg: ["rd.multipath=default","root=/dev/disk/by-label/dm-mpath-root"] |
+| luks_compliant.enabled      | yes      | false                                 | Set it true if you prefer to enable LUKS in ocp deployment |
+| luks_config                 | yes      | []                                    | List of tang servers and thumbprint to apply |
```

Comment:
Could you also add rows for the two required luck_config you have added validation for in the playbook?

Reply:
Added two rows for thumbprint and url which are required in luks_config

Reply:
There are multiple variables with `luks_` prefixed. It will be better to have everything under 1 variable called `luks`. Something similar to:
```
luks:
  enabled: true
  config:
    - thumbprint: ttt
       url: ttttt
  filesystem_device: device
  format: format
  wipeFileSystem: true
  device: dev
  label: label
  options:
    - item1
  wipeVolume: true
  name: name
```

Reply:
Added required changes.